src/tests/Blef.GameLogic.Tests/PokerHands/HighCardTests.cs

```diff
@@ -41,5 +41,16 @@ public void Given_table_with_many_high_cards_Then_result_is_True()
 
             Assert.True(isOnTable);
         }
+
+        [Fact]
+        public void When_comparing_better_hand_to_worse_hand_then_result_is_bigger_than_zero()
+        {
+            HighCard betterHand = new HighCard(Card.Jack);
+            HighCard worseHand = new HighCard(Card.Nine);
+
+            int result = betterHand.CompareTo(worseHand);
```

Comment:
I will introduce: `public enum HandHierarchyResult
{
    Lower,
    Equal,
    Higher
} `

I the enum class name fine?